images/scripts/rhel.setup

```diff
@@ -500,4 +500,7 @@ if [ "${IMAGE#rhel-10*}" != "$IMAGE" ] || [ "${IMAGE#centos-10}" != "$IMAGE" ];
   echo "PerSourcePenalties no" >/etc/ssh/sshd_config.d/99-no-penalties.conf
 fi
 
+# What is wrong with people? "file" is not a valid container log driver.
```

Comment:
Can you please change this to "# HACK" and refer to https://issues.redhat.com/browse/RHEL-68081 , so that we can clean this up at some point?

Reply:
(No need to rebuild the image of course)

Reply:
Yes, that's the plan. :-) But I had to let out some steam first.